gazelle/python/kinds.go

```diff
@@ -32,7 +32,7 @@ func (*Python) Kinds() map[string]rule.KindInfo {
 
 var pyKinds = map[string]rule.KindInfo{
 	pyBinaryKind: {
-		MatchAny: true,
+		MatchAny: false,
```

Comment:
Let's add `MatchAttrs: []string{"srcs"}` so that there aren't two binary targets with the same `srcs` attribute.

Reply:
Done